src/org/traccar/database/PermissionsManager.java

```diff
@@ -75,10 +79,18 @@
         return groupDevices.get(groupId);
     }
 
+    public Set<Long> getUserPermissions(long userId) {
+        if (!userPermissions.containsKey(userId)) {
+            userPermissions.put(userId, new HashSet<Long>());
```

Comment:
I do not think we could

![image](https://cloud.githubusercontent.com/assets/5688080/22007353/e7217704-dc93-11e6-83fa-1e20aa623744.png)